OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs

```diff
@@ -26,9 +26,10 @@ public IngamePowerBarLogic(Widget widget, ModData modData, World world)
 			var powerManager = world.LocalPlayer.PlayerActor.Trait<PowerManager>();
 			var powerBar = widget.Get<ResourceBarWidget>("POWERBAR");
 
+			var powerUsageText = modData.Translation.GetString(PowerUsage) + ": {0}/{1}";
 			powerBar.GetProvided = () => powerManager.PowerProvided;
 			powerBar.GetUsed = () => powerManager.PowerDrained;
-			powerBar.TooltipFormat = modData.Translation.GetString(PowerUsage) + ": {0}/{1}";
+			powerBar.TooltipFormat = powerUsageText;
```

Comment:
```suggestion
			powerBar.TooltipFormat = powerUsage + ": {0}/{1}";
```

Reply:
we do we need to concatenate every frame?

Reply:
`{0}/{1}` could change?

Reply:
https://github.com/OpenRA/OpenRA/pull/20269#discussion_r965586810